### PR TITLE
Serverjs

### PR DESCRIPTION
**Pull request by realswat1**

### CODE REVIEW

File: exposee-app/server.js [hunk 1]
```diff
@@ -1,11 +1,93 @@
...
+      expires: new Date(Date.now() + (365 * 24 * 60 * 60 * 1000)) // 1 year in milliseconds
+    }
+  })
+);
+sessionStore.sync();
+
+app.use(userRoutes);
+
+// Route to get all posts, with associated users
+app.get('/posts', async (req, res) => {
```
Comment (alfonsocheng) [inline exposee-app/server.js:44]: do you need posts for your app?

File: exposee-app/server.js [hunk 2]
```diff
@@ -1,11 +1,93 @@
...
+      order: [['createdAt', 'DESC']]
+    });
+    res.json(posts);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+// Route to create a new post
+app.post('/posts', async (req, res) => {
```
Comment (alfonsocheng) [inline exposee-app/server.js:57]: ditto (do you need posts for your app?)

File: exposee-app/server.js (hunk 1 above)
Comment (realswat1) [reply exposee-app/server.js:44]: in a from of saved streams

**Review by alfonsocheng — APPROVED** [review]
stamping to unblock!

**Pull request merged**